docs/source/assets_writer.rst

```diff
@@ -126,6 +126,7 @@ An example of such a pipeline is in `assets_writer_backpack_2d.lua`_.
 .. _assets_writer_backpack_2d.lua: https://github.com/googlecartographer/cartographer_ros/blob/44459e18102305745c56f92549b87d8e91f434fe/cartographer_ros/configuration_files/assets_writer_backpack_2d.lua
 
 Once you have the ``.ply``, follow the README of `point_cloud_viewer`_ to generate an on-disk octree data structure which can be viewed by one of the viewers (SDL or web based) in the same repo.
+Note that color is required for `point_cloud_viewer` to function.
```

Comment:
If you want inline code highlighting, you need to use double backticks in RST. It's different to Markdown.